src/pages/about-koselugo.astro

```diff
@@ -1,9 +1,82 @@
 ---
+import Section from "../components/Section.astro";
 import Layout from "../layouts/Layout.astro";
 import SimpleHero from "../components/SimpleHero.astro";
+import OneSource from "../components/OneSource.astro";
 ---
 
 <Layout title="About Koselugo">
-    <SimpleHero folder="about-koselugo" alt="" />
-    <div class="wrapper"></div>
-</Layout>
+    <div class="wrapper about-koselugo">
+        <SimpleHero folder="about-koselugo" alt="" />
+        <Section>
+          <Fragment slot="section-title">
+            WHAT IS <span class="color-yellow">KOSELUGO?</span>
+          </Fragment>
+          <div class="wrapper about-koselugo__what-is">
+            <div class="about-koselugo__container about-koselugo__container--flex">
+              <h6 class="about-koselugo__title" >
+                Koselugo is the FIRST and ONLY FDA-approved treatment shown to reduce the size of plexiform neurofibromas (PN)*
+              </h6>
+              <img class="about-koselugo__image" src="/assets/images/about-koselugo/bottles_m.jpg" alt=" ">
+            </div>
+          </div>
+        </Section>
+        <Section noHeader>
+          <div class="wrapper about-koselugo__faq about-koselugo__container about-koselugo__container--visit">
+            <h4 class="about-koselugo__title about-koselugo__title--hide-tablet">Visit the Koselugo Frequently Asked Questions page</h4>
+            <h4 class="about-koselugo__title about-koselugo__title--only-tablet">Do you have questions about Koselugo? Talk to your child's doctor and download our</h4>
+            <a href="https://koselugo.com/faqs" class="button button--basic">Visit the FAQs here</a>
+            <a href="tel:+1888-765-4747" class="button button--basic button--only-mobile">Visit the FAQs here</a>
+          </div>
+        </Section>
+        <Section>
+          <Fragment slot="section-title">
+            CAN KOSELUGO HELP <span class="color-yellow">SHRINK PLEXIFORM TUMORS?</span>
+          </Fragment>
+          <div class="wrapper about-koselugo__clinical-study">
+            <h6 class="about-koselugo__title">KOSELUGO HAS BEEN SHOWN TO SHRINK PLEXIFORM TUMORS IN CHILDREN AGED 2 YEARS AND OLDER</h6>
+            <p>In a clinical study, Koselugo was proven to work for children with NF1 who have plexiform neurofibromas (PN) that cannot be completely removed by surgery:</p>
+            <div class="about-koselugo__container about-koselugo__container--flex">
+              <img class="about-koselugo__image" src="/assets/images/about-koselugo/66-percent-pie-chart.jpg" alt=" ">
```

Comment:
![image](https://user-images.githubusercontent.com/31933833/228381676-f7c8281d-2169-4d37-9602-a36af715dae9.png)

This asset needs max-width. Also probably this content can appears in two columns since 700px.

Like this:
![image](https://user-images.githubusercontent.com/31933833/228381889-8c52be03-12b4-4b32-bddc-4186097be1f2.png)